docs/content/users/usage/cms-settings.md

```diff
@@ -143,3 +143,13 @@ baseVariants:
 ```
 
 See also [TYPO3 Documentation](https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/SiteHandling/BaseVariants.html).
+
+### WordPress Specifics
+
+#### WP-CLI and changing webroot in config.yaml
+If you use something other than the root directory (`''`) for your webroot, wp-cli will not work properly. To fix this, create a `wp-cli.yml` file in the root directory that contains `path: {webroot/path}`.
```

Comment:
```suggestion
If you use something other than the root directory (`''`) for your docroot, wp-cli will not work properly. To fix this, create a `wp-cli.yml` file in the root directory that contains `path: {<docroot>/path}`.
```

Reply:
Don't understand why you marked it resolved without changing the usage from webroot to docroot.